react-navigation/react-navigation
# chore: upgrade yarn to berry

**Motivation**

We're currently on an older version of Yarn. This upgrades Yarn to the latest version.

**Test plan**

- Run the app example app locally
- Run `yarn lerna run prepack` to check if build tasks are working
